cbcontainers/models/compatibility_matrix.go

```diff
@@ -0,0 +1,55 @@
+package models
```

Comment:
I think I would change the file name to `CompatibilityEntry` or something like this.
And maybe add another file to the `AgentVersion`